experimental/packages/otlp-transformer/package.json

```diff
@@ -52,10 +52,13 @@
     "README.md"
   ],
   "peerDependencies": {
-    "@opentelemetry/api": ">=1.3.0 <1.5.0"
+    "@opentelemetry/api": ">=1.3.0 <1.5.0",
+    "@opentelemetry/api-logs": "0.38.0"
   },
   "devDependencies": {
     "@opentelemetry/api": "1.4.1",
+    "@opentelemetry/api-logs": "0.38.0",
+    "@opentelemetry/sdk-logs": "0.38.0",
```

Comment:
`sdk-logs` should be under `dependencies`

Reply:
Only types are imported from `sdk-logs`. It is not necessary.

Reply:
I was looking to be consistent with `sdk-metrics` and `sdk-trace-base`, but not a blocker.

Reply:
It's not as clear as types only => devDependencies because the type checker checks lib types. If the otlp-transformer is used by a tracing-only user they may not have the metrics sdk installed. In that case the type check might fail because the lib check won't find the referenced types. I could be wrong but I think martin is right and it shoudl be in dependencies?

Reply:
You are right. The imported types do appear in the `d.ts` files. In this situation, that dependency should be in `dependencies`.